health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationRedesign/AppConfigurationParentLayer.js

```
@@ -388,6 +165,11 @@ const AppConfigurationParentRedesign = () => {
       isSubmit: stepper?.find((i) => i.active)?.isLast ? true : false,
     });
     if (stepper?.find((i) => i.active)?.isLast) {
+      debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)